data/Language/Lua.yaml

```diff
@@ -0,0 +1,31 @@
+identifier: Lua
+creation_date: 1993
+full_name: Lua
+wikidata: Q207316
+grammar_file: https://github.com/antlr/grammars-v4/blob/master/lua/Lua.g4
+extensions:
+  - lua
+datatypes:
+  - bool_lower_case
```

Comment:
I think that lua have more datatypes 😉

Reply:
Yes, but as I said, other people can add more datatypes as part of the `coAST: Add datatypes to a language definition` GCI task

Reply:
IMHO it would be better if you could at-least mention the datatypes that Lua has so that it becomes easier for the other person to add the datatypes